spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/XsuaaServiceConfigurationDefault.java

```diff
@@ -15,6 +15,8 @@
 @Configuration
 public class XsuaaServiceConfigurationDefault implements XsuaaServiceConfiguration {
 
+	static final String VCAP_SERVICES_CREDENTIALS = "xsuaa credentials from VCAP_SERVICES environment";
```

Comment:
just my humble opinion on this message: when i see this description I need to link it with the NullPointer exception to understand that the xsuaa creds are missing, if i saw message that would have said "xsuaa creds ... are required" I wouldn't have a second thought on what's wrong, so I would suggest to add that "is required" or "must not be empty/null" at the end.

Reply:
I haven't added it as it comes with "NullPointerException", anyhow....